Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -28,6 +28,7 @@
 #import "CGColorSpaceInternal.h"
 #import "CGContextInternal.h"
 #import "CGPathInternal.h"
+#import <CoreGraphics/D2DWrapper.h>
```

Comment:
Given it's not in the same directory as this file.
use <> as per windows documentation. 

---
In reply to: [87101659](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r87101659) [](ancestors = 87101659)